src/adapters/rubicon.js

```diff
@@ -77,6 +83,72 @@ function RubiconAdapter() {
     });
   }
 
+  function buildVideoRequestPayload(bid) {
+    bid.startTime = new Date().getTime();
+
+    let oParams = bid.params;
+
+    if(!oParams || typeof oParams.inventory !== "object") {
```

Comment:
Good catch - should have been oParams.video - need to make sure their is a video object for a video bid